server/command_hooks.go

```diff
@@ -405,6 +417,39 @@ func getDialogWithIntroductionText(introductionText string) model.Dialog {
 	return dialog
 }
 
+func (p *Plugin) executeCommandInteractive(args *model.CommandArgs) *model.CommandResponse {
+	siteURL := *p.API.GetConfig().ServiceSettings.SiteURL
+
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.botId,
+		Message:   "Test interactive button",
+		Props: model.StringInterface{
+			"attachments": []*model.SlackAttachment{{
+				Actions: []*model.PostAction{{
+					Integration: &model.PostActionIntegration{
+						URL: fmt.Sprintf("%s/plugins/%s/interaction/action", siteURL, manifest.Id),
```

Comment:
The `siteURL` is actually not longer needed.

```suggestion
						URL: fmt.Sprintf("/plugins/%s/interaction/action", manifest.Id),
```